provisionerd/provisionerd.go

```diff
@@ -199,6 +201,11 @@ connectLoop:
 			if errors.Is(err, context.Canceled) {
 				return
 			}
+			var sdkErr *codersdk.Error
+			if errors.As(err, &sdkErr) && sdkErr.StatusCode() == http.StatusForbidden {
+				p.opts.Logger.Error(p.closeContext, "failed to dial coderd", slog.Error(err))
```

Comment:
Is it related to this PR or did you discover a bug? A comment on 403 might be helpful.

Reply:
Why log failed to dial instead of not authorized to dial?

Reply:
Those are better words, actually.